gbpservice/nfp/core/cfg.py

```diff
@@ -10,61 +10,57 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-from oslo_config import cfg
 
-es_openstack_opts = [
-    cfg.StrOpt('auth_host',
-               default='localhost',
-               help='Openstack controller IP Address'),
-    cfg.StrOpt('admin_user',
-               help='Admin user name to create service VMs'),
-    cfg.StrOpt('admin_password',
-               help='Admin password to create service VMs'),
-    cfg.StrOpt('admin_tenant_name',
-               help='Admin tenant name to create service VMs'),
-    cfg.StrOpt('admin_tenant_id',
-               help='Admin tenant ID to create service VMs'),
-    cfg.StrOpt('auth_protocol',
-               default='http', help='Auth protocol used.'),
-    cfg.IntOpt('auth_port',
-               default='5000', help='Auth protocol used.'),
-    cfg.IntOpt('bind_port',
-               default='9696', help='Auth protocol used.'),
-    cfg.StrOpt('auth_version',
-               default='v2.0', help='Auth protocol used.'),
-    cfg.StrOpt('auth_uri',
-               default='', help='Auth URI.'),
-]
+from oslo_config import cfg as oslo_config
+
+from gbpservice.nfp.core import version
 
-OPTS = [
-    cfg.IntOpt(
+NFP_OPTS = [
+    oslo_config.IntOpt(
         'workers',
-        default=4,
+        default=1,
         help='Number of event worker process to be created.'
     ),
-    cfg.StrOpt(
-        'modules_dir',
+    oslo_config.StrOpt(
+        'nfp_modules_path',
         default='gbpservice.nfp.core.test',
         help='Path for NFP modules.'
         'All modules from this path are autloaded by framework'
-    ),
-    cfg.IntOpt(
-        'periodic_interval',
-        default=2,
-        help='Interval for event polling task in seconds.'
-        'Polling task wakesup with this interval and'
-        'checks for timedout events.'
-    ),
-    cfg.IntOpt(
-        'reportstate_interval',
-        default=10,
-        help='Interval for report state task in seconds.'
-        'Reporting task will report neutron agents state'
-        'to the plugins at this interval'
-    ),
-    cfg.BoolOpt(
-        'core_debug',
-        default=False,
-        help='Main debug'
     )
 ]
+
+es_openstack_opts = [
```

Comment:
Lets change this name now which has been carried forward from Elastic Services days :)